scripts/run_cpp_tests.py

```diff
@@ -69,7 +69,7 @@
   cmd = 'ctest --output-on-failure -j {0} --schedule-random -R "({1})"'.format(args.j, '|'.join(runtests))
   if args.dry_run:
     print('Dry run requested! Proposed ctest command:', cmd)
-    exit()
+    return
```

Comment:
Seems like this would change the behavior of the script (it was intended to early-exit here).